tools/apollo/create_or_update_organism.py

```diff
@@ -56,7 +101,8 @@
             # mandatory
             genus=args.genus,
             species=args.species,
-            public=args.public
+            public=args.public,
+            blatdb=blat_db
```

Comment:
we should send the blatdb path only if the 2bit index was generated

Reply:
blat_db is set to none if the 2bit was not created, wgich is the default value in both update and create function

Reply:
ah ok nevermind